hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestReplicationSinkManager.java

```diff
@@ -65,12 +90,8 @@ public void testChooseSinks() {
     for (int i = 0; i < totalServers; i++) {
       serverNames.add(mock(ServerName.class));
     }
-
-    when(replicationEndpoint.getRegionServers())
-          .thenReturn(serverNames);
-
+    this.replicationEndpoint.setRegionServers(serverNames);
```

Comment:
nit: I guess 'this' is redundant in multiple places.

Reply:
Will change.